config/sql/migration/migrate.14.sql

```diff
@@ -0,0 +1,21 @@
+-- Don't modify this! Create a new migration instead--see docs/schema-migrations.adoc
+BEGIN TRANSACTION;
+
+CREATE TABLE target_images_data(image_data BLOB NOT NULL);
+CREATE TABLE target_images_info(filename TEXT UNIQUE, real_size INTEGER NOT NULL DEFAULT 0, sha256 TEXT NOT NULL DEFAULT "", sha512 TEXT NOT NULL DEFAULT "", image_id integer NOT NULL,
+        FOREIGN KEY (image_id) REFERENCES target_images_data(rowid) ON DELETE CASCADE
```

Comment:
By reading the sqlite docs (https://www.sqlite.org/foreignkeys.html), does it actually work?

> The parent key is the column or set of columns in the parent table that the foreign key constraint refers to. This is normally, but not always, the primary key of the parent table. **The parent key must be a named column or columns in the parent table, not the rowid**.

and

> Assuming the library is compiled with foreign key constraints enabled, it must still be enabled by the application at runtime, using the PRAGMA foreign_keys command. For example: `sqlite> PRAGMA foreign_keys = ON;`

Reply:
Looks like cascade delete will not work here. But if we don't use here any joins I will just romove foreign key.